Process_Library/GOCART2G_Process.F90

```diff
@@ -1359,10 +1359,10 @@ subroutine Chem_Settling ( km, klid, bin, flag, cdt, grav, &
 
    hsurf => hghte(i1:i2,j1:j2,km)
 
-   allocate(dz(i2,j2,km), radius(i2,j2,km), rhop(i2,j2,km), vsettle(i2,j2,km), qa(i2,j2,km))
+   allocate(dz(i2,j2,km), radius(i2,j2,km), rhop(i2,j2,km), vsettle(i2,j2,km), qa(i2,j2,km), source=0.0)
    allocate(cmass_before(i2,j2), cmass_after(i2,j2))
-   cmass_before = 0.d0
-   cmass_after = 0.d0
+   cmass_before = 0.0_DP
+   cmass_after = 0.0_DP
```

Comment:
Similarly L1363-L1365 can be replaced with:
> allocate(cmass_before(i2,j2), cmass_after(i2,j2),source=0.0_DP)

Reply:
Done. Please take a look